kernel/build-kernel.sh

```diff
@@ -273,7 +273,7 @@ main() {
 		esac
 	done
 
-	shift $(($OPTIND - 1))
+	shift $((OPTIND - 1))
```

Comment:
See this [here](https://unix.stackexchange.com/a/214151).

Do you wish to change it to  ```shift "$((OPTIND-1))"```?

Reply:
@ydjainopensource yeah :) , I think next PR will be to quote a lot of the variables we use :).